uvicorn/server.py

```diff
@@ -28,6 +28,8 @@
     signal.SIGINT,  # Unix signal 2. Sent by Ctrl+C.
     signal.SIGTERM,  # Unix signal 15. Sent by `kill <pid>`.
 )
+if sys.platform == "win32":  # pragma py-linux pragma: py-darwin
```

Comment:
don't we have the negative of win32 or something? :thinking:

Reply:
The negative pragma rule is added in https://github.com/encode/uvicorn/pull/1584/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52, but not yet merged :joy:. I just copy the comment from bellow.

Reply:
Would you mind reviewing that PR as well?

Reply:
I don't have a Windows machine 😢 . I may ask someone else to test it.